apps/web/lib/parseDate.ts

```diff
@@ -46,3 +47,15 @@ export const parseRecurringDates = (
   });
   return [dateStrings, rule.all()];
 };
+
+type BookingItem = inferQueryOutput<"viewer.bookings">["bookings"][number];
+
+export const extractRecurringDates = (
```

Comment:
From a list of bookings, return the list of start date of all of them as dates and as strings.